hybridse/src/udf/default_udf_library.cc

```diff
@@ -666,9 +666,39 @@ void DefaultUdfLibrary::InitStringUdf() {
                 select date_format(date(1590115420000),"%Y-%m-%d");
                 --output "2020-05-22"
             @endcode)");
+    RegisterExternal("upper")
```

Comment:
look like `upper` is symptom to `ucase`, just use `RegisterAlias`

Reply:
`RegisterAlias` cannot generate the doc of another function